### DISCUSSION TITLE
extracting Attachments from a PPTX file

### DISCUSSION THREAD

**Discussion by valerij1975** [ispras/dedoc#386] [Q&A]
Why, when extracting Attachments from a PPTX file, are they not attached anywhere to their location in the document. In docx, a string has a property in Annotation that contains the uid of the next extracted attachment.

**Comment by NastyBoget** [comment]
The feature hasn't been implemented yet for PPTX ([here](https://dedoc.readthedocs.io/en/latest/readers_output/annotations.html) is the list of readers that support this behaviour). We'll do it in the future

**Comment by NastyBoget** [comment]
Added `AttachAnnotation` and `TableAnnotation` to `PPTX` in [dedoc==2.0](https://pypi.org/project/dedoc/2.0/)
